lib/pengine/bundle.c

```diff
@@ -78,9 +78,10 @@ typedef struct pe__bundle_variant_data_s {
         enum pe__container_agent agent_type;
 } pe__bundle_variant_data_t;
 
-#define get_bundle_variant_data(data, rsc)                              \
-    CRM_ASSERT(pcmk__is_bundle(rsc) && (rsc->variant_opaque != NULL));  \
```

Comment:
Doesn't hurt to continue asserting on `variant_opaque` in case a resource is ever mistakenly created outside of the unpacking code. That's especially paranoid so whatever.

Reply:
I decided against it because there's no clear stopping line -- why not check every member for valid values? A bunch of other members should always be non-NULL (and are assumed as such at various points).